core/http-auth-spi/src/main/java/software/amazon/awssdk/http/auth/spi/SyncSignRequest.java

```diff
@@ -44,6 +46,6 @@ static <IdentityT extends Identity> Builder<IdentityT> builder(IdentityT identit
      */
     interface Builder<IdentityT extends Identity>
         extends SignRequest.Builder<Builder<IdentityT>, ContentStreamProvider, IdentityT>,
-                SdkBuilder<Builder<IdentityT>, SyncSignRequest<IdentityT>> {
+            CopyableBuilder<Builder<IdentityT>, SyncSignRequest<IdentityT>> {
```

Comment:
Instead of `this.properties = request.properties;` in the builder's constructor, it could call `this.properties.putAll(request.properties)`. This would also let you not remove the `final` from the parent builder's properties member.